frontend/src/plugins/__tests__/extensions.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { Extension } from '@openshift/dynamic-plugin-sdk';
+import extensionDeclarations from '~/plugins/extensions';
+import { isCodeRef } from '~/plugins/internal/coderefs';
+import { visitDeep } from '~/plugins/internal/objects';
+
+describe('extensions', () => {
+  it('should be valid', () => {
+    expect(validateExtensions(extensionDeclarations)).toBe(true);
+  });
+});
+
+// Tests the test function
+describe('validateExtensions', () => {
+  it('should validate a valid code ref', () => {
+    expect(
+      validateExtensions([
+        {
+          type: 'test',
+          properties: {
+            firstLevel: {
+              props: {
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
+                content: () => import('./Test'),
```

Comment:
This looks weird -- tests using `./Test` -- but then it's not a test file -- it's just a placeholder. Looking at it in my IDE is definitely better -- but I worry it's a weird optics for a test file

Reply:
That's a fair point. Something like `ExampleComponent` is probably a better name to use.